code-challenges.test.js

```diff
@@ -14,39 +14,103 @@
 
 // a) Create a test with expect statements for each of the variables provided.
 
-const numbersArray1 = [6, 7, 8, 9, 10]
-// Expected output: [18, 21, 24, 27, 30]
-const numbersArray2 = [24, 27, 30, 33, 36]
-// Expected output: [72, 81, 90, 99, 108]
-
+describe ("multiplyByThree", () => {
+    it ("is a function that takes in an array of numbers and returns an array with all the numbers multiplied by 3", () => {
+        const numbersArray1 = [6, 7, 8, 9, 10]
+        // Expected output: [18, 21, 24, 27, 30]
+        const numbersArray2 = [24, 27, 30, 33, 36]
+        // Expected output: [72, 81, 90, 99, 108]
+        expect(multiplyByThree(numbersArray1)).toEqual([18, 21, 24, 27, 30]) 
+        expect(multiplyByThree(numbersArray2)).toEqual([72, 81, 90, 99, 108])
+    })
+})
+const multiplyByThree = (array) => {
+    return array.map((value) => {
+    return value * 3
+    })
+}
 // b) Create the function that makes the test pass.
 
 // Pseudo code:
+// input: single array of numbers
+// output: array with the numbers mult by 3
+// do something to every item in the array - map
+// value * 3
+
 
 // --------------------1) Create a function that takes a object as an argument and decides if the number inside it is evenly divisible by three or not.
 
 // a) Create a test with expect statements for each of the variables provided.
 
-const object1 = { number: 15 }
-// Expected output: "15 is divisible by three"
-const object2 = { number: 0 }
-// Expected output: "0 is divisible by three"
-const object3 = { number: -7 }
-// Expected output: "-7 is not divisible by three"
+describe ("divByThree", () => {
+    it ("is a function that takes a object as an argument and decides if the number inside it is evenly divisible by three or not", () => {
+        const object1 = { number: 15 }
+        // Expected output: "15 is divisible by three"
+        const object2 = { number: 0 }
+        // Expected output: "0 is divisible by three"
+        const object3 = { number: -7 }
+        // Expected output: "-7 is not divisible by three"
+        expect(divByThree(15)).toEqual("15 is divisible by three") 
+        expect(divByThree(0)).toEqual("0 is divisible by three")
+        expect(divByThree(-7)).toEqual("-7 is not divisible by three")
+    })
+})
```

Comment:
Your test structure looks great, but you aren't actually using your test variables so you are coding for the wrong input.